### DISCUSSION TITLE
Having Create React App SPA and NextJS SSR under same domain

### DISCUSSION THREAD

**Discussion by jmromeroeci** [vercel/community#304] [Help] [answered]
Hi! I'm doing an incremental migration from a Crate React SPA to a Next SSR app. I would like to have them both under the same domain and only route them depending on the URL. For example, everything under `app.example.com/marketplace` should be redirected to the SSR server and everything else (`app.example.com/*`) should be redirected through the SPA.
I saw some examples in the [monorepos post](https://vercel.com/blog/monorepos) but they're not sharing the domain, rather they're subdomains. 
Is it possible to configure some sort of proxy to redirect based on URL?

Kind regards

**Comment by mcsdevv** [comment, marked as answer]
Hey @jmromeroeci!

One thing to check here would be the [Migrating to Next.js guide](https://nextjs.org/docs/migrating/incremental-adoption).
